lib/utils/getAmountWithCurrency.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export const getAmountWithCurrency = (locale, currency, amount) => (
+  Intl.NumberFormat(locale, { style: 'currency', currency }).format(amount)
```

Comment:
shouldn't you set any default values?

Reply:
still, shouldn't it be like `amount = 0` ? in case someone calls this function directly

Reply:
yep. forgot about this case. thank you)